letsencrypt-auto-source/pieces/fetch.py

```diff
@@ -112,6 +120,17 @@ def verified_new_le_auto(get, tag, temp_dir):
                             "certbot-auto.", exc)
 
 
+def create_CERT_NONE_context():
+    """Create a SSLContext object to not check hostname."""
+    # PROTOCOL_TLS isn't available before 2.7.13 but this code is for 2.7.9+, so use this.
+    context = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+
+    context.check_hostname = False
```

Comment:
nit: I don't think this line is necessary if `verify_mode` is set to `CERT_NONE`.